packages/kbn-plugin-generator/sao_template/template/public/index.ts

```diff
@@ -0,0 +1,15 @@
+<%_ if (hasScss) { -%>
+import './index.scss';
+<%_ } -%>
+
+import { <%= upperCamelCaseName %>Plugin } from './plugin';
+
+// These exports are the public contract for your static code and types.
```

Comment:
nit: maybe we should not mention "static" as when the `plugin()` is initialized by the platform it is stateful.

```suggestion
// This exports static code and TypeScript types, as well as, Kibana Platform `plugin()` initializer.
```